target/linux/realtek/dts/rtl8380_zyxel_gs1900-8.dts

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "rtl8380_zyxel_gs1900.dtsi"
+
+/ {
+	compatible = "zyxel,gs1900-8", "realtek,rtl838x-soc";
+	model = "ZyXEL GS1900-8 v1 Switch";
```

Comment:
Well, either it's v1, then this should be added to device name/compatible. Or it isn't, then remove it in the model as well, please.

Reply:
Changed as requested, with the explanation below.